### DISCUSSION TITLE
Java version for MOSAIC

### DISCUSSION THREAD

**Discussion by DFCui** [eclipse/mosaic#212] [Q&A] [answered]
Hello,

As stated in MOSAIC documentation and README.md, a Java runtime environment needs to be installed.  I have a question about the Java version that MOSAIC supports.
In the README.md (https://github.com/eclipse/mosaic), either Java 8 or 11 can be installed.
![image](https://user-images.githubusercontent.com/24232678/168402027-aa7cc832-879a-4c47-9a95-5ed520263d2b.png)

However, in the MOSAIC documentation (https://www.eclipse.org/mosaic/docs/getting_started/), only Java 8 is supported and Java 11 or above is limited supported, as shown in the following screen capture. https://www.eclipse.org/mosaic/docs/getting_started/
![image](https://user-images.githubusercontent.com/24232678/168402094-74aaf04e-d50a-4eae-8b08-dbd7b7c75255.png)

Does anyone know whether Java version 11 is fully supported or not? Can I use Java 11 for building and running MOSAIC? 

Thank you very much.

**Comment by kschrab** [comment, marked as answer]
Hi, I think everything will work with Java 11 and above. We do not test MOSAIC extensively with higher versions of Java and therefor suggest to use Java 8. Just try out and report any issues you potentially find.

**Reply by DFCui** [reply]
Thank you for your prompt reply.
